### PR TITLE
Remove functions from sqitch

### PR DESCRIPTION
**Pull request by MagneDavidsen**
It feels like Sqitch isn't really made for altering functions.

Because changes to functions result in copying the whole function to a new file (_versionX) and make change to it, it is impossible to follow the changes in git. It also results in a lot of files, which makes  are impossible to follow in git.

This is a proposal to remove functions from _sqitch_, and instead just keep them in sql-files that we can deploy the old school way.

### CODE REVIEW

File: functions/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+     *)
+         echo "No environment specified"
+         exit 3
+         ;;
+esac
+
+for f in *.sql
+do
+ echo "deploying $f to $HOST"
+ psql -f $f -h $HOST -d floq -U root
```
Comment (ehamberg) [inline functions/deploy.sh]: You should double quote `$f` to prevent word splitting or globbing.

Comment (hestad) [inline functions/deploy.sh]: https://www.shellcheck.net/

**Comment by martingms** [comment]
Remove functions from _sqitch_, I presume? What's the plan with triggers etc, are they put with functions or in sqitch?

**Comment by MagneDavidsen** [comment]
@martingms so far i have kept a few functions in sqitch, because we have views that depend on them. i guess we maybe should keep triggers in sqitch as well, as they tend to be tightly connected to the tables?

i guess the problem with functions in sqitch mainly comes from that we actually use them to create the api, because postgrest. this leads to more functions, and more changes to functions, than we would have had if we didn't use postgrest. it seems?

File: functions/deploy.sh (hunk 1 above)
Comment (MagneDavidsen) [inline functions/deploy.sh]: Done

**Comment by martingms** [comment]
@MagneDavidsen well, views are also typically recreated on changes... I guess one extreme is having only tables (and custom datatypes) in sqitch, and perhaps indices, and everything else outside. Personally I'm leaning towards that right now, but haven't really thought it through well enough.

**Comment by MagneDavidsen** [comment]
@martingms sounds good to me. the actual difference right now is just to exclude absensence_view.sql from sqitch as well?

**Comment by hestad** [comment]
Keeping idempotent structures or at least those we recreate sounds reasonable to me.

**Comment by hestad** [comment]
Oops :p

**Pull request merged**